src/assets/translations/en/main.json

```diff
@@ -607,25 +607,22 @@
       "thorchain": {
         "deposit": "Deposit",
         "depositRefund": "Deposit Refund",
+        "depositRefundNative": "Deposit + Refund",
```

Comment:
Thorchain native transactions that include both the inbound and outbound transactions